lib/react_on_rails/helper.rb

```diff
@@ -279,8 +279,9 @@ def rails_context(server_side: true)
           # URLs as UTF-8. This situation can occur in browsers that do not encode the
           # entire URL as UTF-8 already, mostly on the Windows platform (IE11 and lower).
           original_url_normalized = request.original_url
-          if original_url_normalized.encoding.to_s == "ASCII-8BIT"
-            original_url_normalized = original_url_normalized.force_encoding("ISO-8859-1").encode("UTF-8")
+          if original_url_normalized.encoding == Encoding::BINARY
+            original_url_normalized.force_encoding(Encoding::ISO_8859_1)
+            original_url_normalized = original_url_normalized.encode(Encoding::UTF_8)
```

Comment:
@casperisfine any reason to put this on 2 separate lines?

Reply:
To reduce line length.

Reply:
Please keep as one line. The line is not long.

Reply:
Done.

Reply:
Thanks.

Reply:
```
lib/react_on_rails/helper.rb:283:121: C: Layout/LineLength: Line is too long. [122/120]
            original_url_normalized = original_url_normalized.force_encoding(Encoding::ISO_8859_1).encode(Encoding::UTF_8)
```

Reply:
My bad!

```
Inspecting 11[5](https://github.com/shakacode/react_on_rails/runs/5315703112?check_suite_focus=true#step:10:5) files
......................C............................................................................................

Offenses:

lib/react_on_rails/helper.rb:2[8](https://github.com/shakacode/react_on_rails/runs/5315703112?check_suite_focus=true#step:10:8)3:121: C: Layout/LineLength: Line is too long. [122/120]
            original_url_normalized = original_url_normalized.force_encoding(Encoding::ISO_885[9](https://github.com/shakacode/react_on_rails/runs/5315703112?check_suite_focus=true#step:10:9)_1).encode(Encoding::UTF_8)
```

How about just breaking into 2 lines with the dot on the end of the line?

Reply:
Done.